### DISCUSSION TITLE
WLAN n/a or unstable on Shelly2pm pro running tasmota 13.2.0

### DISCUSSION THREAD

**Discussion by SailorMax59** [arendst/Tasmota#20219] [Support request]
Hello,
I’m running some 20 Shelly 2PM (ESP32-U4WDH-D rev.3) for controlling my shutters. Some of them (currently 3) needed to be reflashed.
I’m using esptool and have flashed tasmota32solo1.factory.bin (13.2.0). Tasmota32solo1.bin can also be programmed, but will not start the AP mode. The “Tasmota32solo1.factory.bin” version will start in AP mode, e.g. “tasmota-687DE-7652” is visible in WLAN. Sometimes I was able to connect to the device (192.168.4.1) and change the WLAN credentials. Most times, I will not be able to connect  (192.168.4.1). 
After restart, the “good” device was visible in my home WLAN, got an IP assigned and I could connect to it. After running the “Autoconfig” to Shelly 2PM, the device is dead (no more visible in my WLAN, not falling back to tasmota default).

Any ideas?

Regards Matthias

**Comment by sfromis** [comment]
Try removing the flashing setup, and instead power via mains. The current needed for Wifi is often not available from some USB/serial adapter.

**Reply by SailorMax59** [reply]
I've flashed via USB. Further work is done on mains supply
